label_studio_converter/brush.py

```diff
@@ -38,6 +38,18 @@
 
```

Comment:
Also we have to add brushlabels to supported format:
https://github.com/heartexlabs/label-studio-converter/pull/175/files#diff-322ce95b5aa5f02fb330c7f62be1ea0066c01829b564db0644872e09338a3ddfL309
check this line ^